docs/architecture/modernize-desktop/whats-new-with-netcore3- for-desktop.md

```diff
@@ -0,0 +1,147 @@
+# What&#39;s new with .NET Core 3.0 for Desktop?
```

Comment:
- Why there is no metadata header for this file ?
- This file name contains a space, it should be removed.
- Why encode thr `'` character ? I don't think there is a problem putting it directly.

   ```suggestion
   # What's new with .NET Core 3.0 for Desktop?
   ```